package.json

```diff
@@ -57,7 +57,9 @@
     "@storybook/ui": "6.1.18",
     "@testing-library/svelte": "^3.0.3",
     "@tsconfig/svelte": "^1.0.10",
+    "@types/fs-extra": "9.0.12",
     "@types/jest": "^26.0.22",
+    "@types/luxon": "^2.0.5",
```

Comment:
does adding packages make our bundle too big?

Reply:
Not sure, but luxon is necessary for this feature. The other packages I looked at with timezone offsets weren't as robust, and would've been buggy.